cmd/relayer/main.go

```diff
@@ -64,6 +72,27 @@ var (
 			Value: "info",
 			Usage: "Set log level: one of [error|warn|info|debug]",
 		},
+		&cli.BoolFlag{
+			Name:  flagWithNetwork,
+			Value: false,
+			Usage: "Run the relayer with p2p network capabilities",
+		},
+		&cli.StringFlag{
+			Name:  flagLibp2pKey,
+			Usage: "libp2p private key",
+			Value: fmt.Sprintf("%s/node.key", os.TempDir()),
```

Comment:
Can we do something similar to what we do with the atomic swap files? (A dot directory in the user's home.) The top of `/tmp` is shared by all the users on a system (another user could be occupying the file name), it is cleared on reboot, and it doesn't play nice with docker, where the user will want this file and others on an external (to the container) volume.

Reply:
I don't expect this binary to be actually used, it's more of an example app for this repo. I plan to implement a different binary for the swap into the swap repo